tests/FakeItEasy.Analyzer.CSharp.Tests/UnusedReturnValueAnalyzerTests.cs

```diff
@@ -359,7 +359,7 @@ class TheClass
         void Test()
         {
             var fake = A.Fake<IFoo>();
-            A.CallTo(fake).Where(call => true, output => new object());
+            A.CallTo(fake).Where(call => true, null);

```

Comment:
For consistency with the corresponding VB test that had to be changed after enabling `Option Strict`